presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -63,13 +63,23 @@ export class QueryListItem extends React.Component {
         }
 
         let formattedQueryText = "";
+        let currentLine = 0;
```

Comment:
Let's rename it into something like `currentLineCount`.

Reply:
done